mon-pix/package.json

```diff
@@ -112,5 +112,9 @@
     "showdown": "^1.9.1",
     "stylelint": "^13.6.1",
     "stylelint-config-standard": "^20.0.0"
+  },
+  "dependencies": {
+    "@formatjs/intl-getcanonicallocales": "^1.4.6",
+    "@formatjs/intl-pluralrules": "^3.4.10"
```

Comment:
**question (blocking):** Pourquoi ce ne sont pas des `devDependencies` ?
https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies

Reply:
Merci pour ce retour avec les explications détaillées, je pense que j'avais toujours compris de travers le : 
`dependencies are required to run, devDependencies only to develop`

Reply:
Je pense que ça va plutôt dans `dependencies` car nous avons besoin de ces fichiers au runtime (et pas pour le développement).

Reply:
Je ne suis pas certain qu'on en ait besoin au runtime.
D'ailleurs ember-intl est une devDependancy

Reply:
Ok en creusant plus loin j'ai trouvé ça : 
> Because this is a polyfill (which will run before your source code), we need it to be a dependency, not a devDependency (https://babeljs.io/docs/en/babel-polyfill)

>devDependencies should contain packages which are used during development or which are used to build your bundle, for example, mocha, jscs, grunt-contrib-watch, gulp-jade and etc. These packages are neseccery only while you are developing your project, also ESlint is used to check everything during building your bundle. (https://medium.com/@stalonadsl948/dependencies-vs-devdependencies-926e096a3dee)

Ça semble donc bien être une `dependancy`